src/apps/loyalty/containers/thank_you/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react"
+
+import colors from "../../../../assets/colors"
+import Button from "../../../../components/buttons/inverted"
+import Nav from "../../../../components/nav"
+import NavItem from "../../../../components/nav_item"
+import Text from "../../../../components/text"
+import TextLink from "../../../../components/text_link"
+import Title from "../../../../components/title"
+
+export default props => (
+  <div style={{textAlign: "center"}}>
+    <Nav>
+      <NavItem href="https://www.artsy.net">Back To Artsy</NavItem>
+    </Nav>
+
+    <div style={{marginTop: "40px"}}>
```

Comment:
yeah I'd suggest using `styled-components` for this container div. We can include @media-queries in there for different screen-size support